### DISCUSSION TITLE
일반 개발자도 맵 업로드가 가능할까요?

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#38] [5. Q&A]
1. 저는, 심사에서 거절되었던 캐릭터생성문제가 해결되어서 심사제출을 해도 되는걸까요?
2. 11월 26일에 제출해서 심사거절되었던 캐릭터 생성문제가 해결되었을까요?

**Comment by cheolhui-lee** [comment]
안녕하세요.
캐릭터 관련 이슈는 해결이 되어서 심사 제출을 진행해주셔도 될 것 같습니다!
혹시 업로드나 심사 제출 버튼의 동작 관련하여 이슈가 있다면 또 말씀 부탁드립니다.
고맙습니다.
